### ISSUE TITLE
License memote report under CC-BY

### ISSUE THREAD

**Issue by matthiaskoenig** [opencobra/memote#764]
Hi everyone,

I've recently encountered an issue regarding the legal constraints of including a memote report in the supplementary materials of a publication. The primary concern is that the report is not licensed under CC-BY.

The specific issues are as follows:
1. **Lack of Clear Licensing Statement**: There is no explicit statement regarding the license of the report. While it mentions "Copyright (c) 2017, Novo Nordisk Foundation Center for Biosustainability, Technical University of Denmark" and a code license under the [Apache Software License 2.0](https://memote.readthedocs.io/en/latest/LICENSE), this does not permit publication under CC-BY. A clear statement should be added, such as: "The memote report is licensed under Creative Commons (CC) 4.0." As it stands, the report cannot be distributed with publications or on webpages without permission from the Novo Nordisk Foundation Center for Biosustainability, Technical University of Denmark, which is contrary to the intended purpose.

2. **Use of Protected Logos**: The report includes logos, specifically the GitHub logo, which are likely protected by copyright. Either these logos should be removed, or it should be explicitly stated that they are under CC-0 (which is unlikely).

Resolving these issues is important as it currently creates uncertainty about how the report can be distributed. For instance, even adding the report to a webpage or GitHub alongside a model could result in copyright problems.

Best regards,
Matthias

**Comment by Midnighter** [comment]
Thanks for bringing this up Matthias. It indeed was never our intention to restrict use of the reports in this way. Could you make a PR that changes the report to CC-BY? I simply don't have capacity for that at the moment.

I'm less clear about the GitHub logo. My understanding of https://github.com/logos is that if you use the logo to link to a repo on GitHub that is allowed. And indeed it is used by countless sites to do just that.

**Comment by matthiaskoenig** [comment]
The linking is allowed, but that does not make the image CC-BY. I.e. the copyright is clearly the following for the logos:
> GITHUB®, the GITHUB® logo design, the INVERTOCAT logo design, OCTOCAT®, and the OCTOCAT® logo design are trademarks of GitHub, Inc., registered in the United States and other countries.
The OCTOCAT design is the exclusive property of GitHub, Inc and has been federally registered with the United States Copyright Office. All rights reserved.
No adaptation or use of any kind of any of our registered trademarks or copyrights, or any other contents of this website, is allowed without the express written permission of GitHub, Inc.
For more information regarding the authorized uses of these items, please [contact us](https://github.com/contact).

I.e. you can link to github via the logo, but you cannot distribute anything with the logo as CC-BY because the octocat is not CC-BY licensed.
I.e. to distribute the report as CC-BY we have to remove the github logo from the report.
I will make a pull request for the clarification and removal of logo during the day.

Basically this came up via the Plos Comp Bio copyright people who clearly stated they cannot publish this with the logo under CC-BY in a supplement or article.

**Comment by draeger** [comment]
Question: Would it be possible to include the logo as a refernce, i.e., not within the report but linked via an HTML image refernence? In this way, the report would look exactly the same but the logo would be loaded from the internet when opening the document.
